libraries/ESP8266WiFi/src/include/UdpContext.h

```diff
@@ -265,7 +290,6 @@ class UdpContext
             // ref'ing it to prevent release from the below pbuf_free(deleteme)
             pbuf_ref(_rx_buf);
         }
-        // remove the already-consumed head of the chain
```

Comment:
Perhaps it has to be replaced by
"remove parts of the pbuf chain that are not anymore "ref"ed